postgres/postgres.go

```diff
@@ -79,6 +79,14 @@ func (s *Stmt) Prepare(db *sqlx.DB, SQL string) (statement *sql.Stmt, err error)
 // Load postgres
 func Load() {
 	config.PrestConf.Adapter = &Postgres{}
+	db, err := connection.Get()
```

Comment:
I know that today we do not have test for `Load()`: It is important to write test for your new contribution

Reply:
@avelino  done!